src/Console/Application.php

```diff
@@ -160,4 +164,37 @@ protected function getDefaultCommands(): array
     {
         return [new HelpCommand(), new ListCommand()];
     }
+
+    /**
+     * @throws \Throwable
+     */
+    protected function doRunCommand(Command $command, InputInterface $input, OutputInterface $output): int
+    {
+        $this->executedCommand = $command;
+
+        return parent::doRunCommand($command, $input, $output);
+    }
+
+    protected function doRenderThrowable(\Throwable $e, OutputInterface $output): void
+    {
+        // Since parallel analysis utilises child processes, and they have their own output,
+        // we need to capture the output of the child process to determine it there was an exception.
+        // Default render format is not machine-friendly, so we need to override it for `worker` command,
+        // in order to be able to easily parse exception data for further displaying on main process' side.
+        if ($this->executedCommand instanceof WorkerCommand) {
```

Comment:
I am not sure, because it's the `Application` which runs the `Command` and handles exceptions thrown there. We could add `try/catch` inside `WorkerCommad::execute()`, but the current approach will catch more errors, also coming from command's initialisation etc.